Directory.Build.props

```diff
@@ -37,6 +37,7 @@
     -->
     <AndroidPackVersion>35.0.0</AndroidPackVersion>
     <AndroidPackVersionSuffix>preview.7</AndroidPackVersionSuffix>
+    <IsStableBuild>false</IsStableBuild>
```

Comment:
I was comparing what we have on the `release/8.0.2xx` branch:
https://github.com/dotnet/android/blob/36bbac0b3032e6ad566060d42f891a58db027836/Directory.Build.props#L32-L41

Could this check `$(AndroidPackVersionSuffix) == 'rtm'` to determine true or false?

Reply:
That sounds reasonable to me, I think it would be best to not introduce more properties that need to be manually tweaked when stablizing.

I was looking at `AndroidPackVersionSuffix` and I think it's not used anywhere when moving to stable package versions? I don't think that's an issue, we'll just have to make sure to still update it to rtm each time even though it won't be referenced by anything?